tests/tracer.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+
+	"github.com/weaveworks/tcptracer-bpf/pkg/event"
+	"github.com/weaveworks/tcptracer-bpf/pkg/tracer"
```

Comment:
In fact, what's the reason for having multiple packages? In most (if not all) cases there's a single module per package. That's not very idiomatic AFAIK.